lib/couchdb/index.js

```diff
@@ -119,6 +120,9 @@ const findById = R.curry((couch, bucket, projection, id) => R.composeP(
 )(bucket, id))
 
 
+const del = R.curry((couch, bucket, id, rev) => couch.del(bucket, id, rev))
```

Comment:
noticed the MemoryStore.deleteById returns true on successful delete. This one should also return true in the same case